treeherder/log_parser/parsers.py

```diff
@@ -403,7 +393,7 @@ class ErrorParser(ParserBase):
     RE_EXCLUDE_1_SEARCH = re.compile(r"TEST-(?:INFO|PASS) ")
 
     RE_EXCLUDE_2_SEARCH = re.compile(
-        r"I[ /](Gecko|Robocop|TestRunner).*TEST-UNEXPECTED-"
```

Comment:
Robocop is still active on esr-68 (only), if that's a concern. However, I don't recall seeing that format of error -- it may be obsolete anyway.

Reply:
Nowadays this rule [only duplicates error lines](https://treeherder.mozilla.org/logviewer.html#?job_id=292924606&repo=mozilla-esr68) as far as I can see.